Dockerfile

```diff
@@ -1,9 +1,25 @@
-FROM scratch
+FROM golang:1.14-buster AS build
+
+ENV GOBIN=$GOPATH/bin
+
+ADD . /src/estafette-gke-preemptible-killer
+
+WORKDIR /src/estafette-gke-preemptible-killer
+
+RUN apt-get -qqq update \
+    && apt-get -qqq -y install ca-certificates\
+    && update-ca-certificates
+
+RUN go mod download \
+    && go build ./...
+
+FROM debian:buster-slim
```

Comment:
The runtime image should still be `scratch` in order to reduce attack surface and size.

Reply:
I could not make it run with scratch. I figure out that for some reason does not like the 

> COPY --from=build /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/